storm-core/src/jvm/backtype/storm/multilang/JsonSerializer.java

```diff
@@ -33,20 +33,24 @@
 import org.json.simple.JSONValue;
 
 import backtype.storm.task.TopologyContext;
-import backtype.storm.tuple.Tuple;
 import backtype.storm.utils.Utils;
 
 /**
  * JsonSerializer implements the JSON multilang protocol.
  */
 public class JsonSerializer implements ISerializer {
-    private DataOutputStream processIn;
+    //ANY CHANGE TO THIS CODE MUST BE SERIALIZABLE COMPATIBLE OR THERE WILL BE PROBLEMS
+    private static final long serialVersionUID = 8581179559440582569L;
+
+    public static final String DEFAULT_CHARSET = "UTF-8";
+
+    private BufferedWriter processIn;
     private BufferedReader processOut;
```

Comment:
If we are setting the serialVersionUID (which I think is great) can we mark processIn and processOut as transient?